doc/github_handler.rst

```diff
@@ -0,0 +1,60 @@
+GitHub Handler
+==============
+
+GitHub Handler is a Universum mode that serves as GitHub Application, helping to perform and report checks on
+new commits to a repository. In this mode Universum can create new check runs on GitHub and trigger an already
+set up automation server to perform these checks. GitHub Handler parses all required params and passes them
+to the triggered builds.
+
+
+How to set up GitHub auto check using Unversum
+----------------------------------------------
+
+Universum in default, 'main' mode when set up to ``--vcs-type=github`` and with ``--report-to-review`` passed
```

Comment:
This sentence is not grammatically correct.
I think we can split it and make more clear.